site/docs/tutorial/ios-app.md

```diff
@@ -246,6 +266,9 @@ The `.ipa` file and other outputs are located in the
 ### Run and debug the app in the simulator
 
 You can now run the app from Xcode using the iOS Simulator. First, [generate an Xcode project using Tulsi](http://tulsi.bazel.io/).
+
+**TODO:** Better documentation on using Tulsi
```

Comment:
Let's not add "Todo's" to customer-facing docs.

Reply:
Fixed